packages/extension-task-item/package.json

```diff
@@ -23,6 +23,9 @@
   "peerDependencies": {
     "@tiptap/core": "^2.0.0-beta.1"
   },
+  "dependencies": {
+    "prosemirror-model": "^1.16.1"
```

Comment:
Is this dependency needed or should this be a peerDependency?

Reply:
Sure, just needed for the types

Reply:
alright, thanks for the quick answer!